docs/modelserving/v1beta1/amd/README.md

```diff
@@ -0,0 +1,189 @@
+This example demonstrates how to deploy a Tensorflow GraphDef model on KServe with the AMD Inference Server to run inference on AMD EPYC CPUs.
+
+## Prerequisites
+
+These example was tested on an Ubuntu 18.04 host machine using the Bash shell.
+
+These instructions assume:
+- You have a machine with a modern version of Docker (>=18.09) and sufficient disk space to build the image
+- You have a Kubernetes cluster set up
+- KServe has been installed on the Kubernetes cluster
+- Some familiarity with Kubernetes / KServe
+
+Refer to the installation instructions for these tools to install them if needed.
+
+## Set Up the Image
+
+This example uses the [AMD ZenDNN](https://developer.amd.com/zendnn/) backend to run inference on TensorFlow models on AMD EPYC CPUs.
+To build a Docker image for the AMD Inference Server that uses this backend, download the `TF_v2.9_ZenDNN_v3.3_C++_API.zip` package from ZenDNN.
+You must agree to the EULA to download this package.
+You need a modern version of Docker (at least 18.09) to build this image.
+
+```bash
+# clone the inference server repository
+git clone https://github.com/Xilinx/inference-server.git
+
+# place the downloaded ZenDNN zip in the repository
+mv *.zip ./inference-server/
+
+# build the image
+cd inference-server
+./proteus dockerize --production --tfzendnn=./TF_v2.9_ZenDNN_v3.3_C++_API.zip
+```
+
+This builds an image on your host: `<username>/proteus:latest`.
+This image can now be uploaded to a [local Docker registry server](https://docs.docker.com/registry/deploying/) to use with KServe.
+You will need to update the YAML files in this example to use this image.
+
+More documentation for building a ZenDNN image for KServe is available: [ZenDNN + Inference Server](https://xilinx.github.io/inference-server/main/zendnn.html) and [KServe + Inference Server](https://xilinx.github.io/inference-server/main/kserve.html).
+
+## Set up the model
+
+In this example, you will use an [MNIST Tensorflow model](https://github.com/Xilinx/inference-server/blob/main/tests/assets/mnist.zip).
+
+To use your own models, you first need a GraphDef model to serve.
+One way to do this is to [freeze a checkpoint](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/freeze_graph.py).
+Once you have the model, you also need a `config.pbtxt`, which defines some metadata about your model such as its name and input/output tensors.
+The `config.pbtxt` for our example MNIST model is the following:
+
+```
+name: "mnist"
+platform: "tensorflow_graphdef"
+inputs [
+  {
+    name: "images_in"
+    datatype: "FP32"
+    shape: [28,28,1]
+  }
+]
+outputs [
+  {
+    name: "flatten/Reshape"
+    datatype: "FP32"
+    shape: [10]
+  }
+]
+```
+
+The name, platform, inputs and outputs must all be defined.
+Some notes about the acceptable values:
+- the name must uniquely identify a model for the server
+- the platform must be one of `tfzendnn_graphdef` or `vitis_xmodel`
```

Comment:
can you elaborate what's `vitis_xmodel` ?

Reply:
XModel is the format used by Xilinx's [Vitis AI](https://github.com/Xilinx/Vitis-AI) for FPGAs. There are a number of pre-compiled models available [online](https://github.com/Xilinx/Vitis-AI/tree/master/model_zoo). But I'll take it out here since these instructions are meant for CPUs.